tests/test_chiral_restraints.py

```diff
@@ -289,7 +293,8 @@ def test_chiral_restraints_tetrahedral():
     U_fn = system.get_U_fn()
 
     vols_orig = []
-    frames = simulate_system(U_fn, x0)
+    # turn off minimizing to avoid accidentally swapping chiral states
+    frames = simulate_system(U_fn, x0, minimize=False)
```

Comment:
Can you comment on the lack of minimization? Shouldn't the `U_fn` include the chiral restraints?